src/CMakeLists.txt

```diff
@@ -86,11 +86,19 @@ endif()
 # link libraries and header which are then excluded from runtime component
 set (CMAKE_INSTALL_DEFAULT_COMPONENT_NAME "xrt")
 
-# Enable development package by specifying development component name
+# Enable development package by setting XRT_DEV_COMPONENT to "xrt-dev"
 # If XRT_DEV_COMPONENT is same DEFAULT_COMPONENT then only that package
 # is created with both development and run-time content.
 #set (XRT_DEV_COMPONENT "xrt-dev")
-set (XRT_DEV_COMPONENT "xrt")
+set (XRT_DEV_COMPONENT "xrt" CACHE STRING "Set to 'xrt-dev' to split dev artifacts into separate component&package")
```

Comment:
The logic CPack uses for it's configuration variables means that if you set `XRT_DEV_COMPONENT` to something other than the default `"xrt"` or `"xrt-dev"`, your `XRT_DEV_COMPONENT` package won't have the correct settings because cpackLin.cmake sets `CPACK_<PKG_TYPE>_XRT-DEV_*` variables and I didn't want to figure out it if it is possible to do something like:
`SET(CPACK_RPM_${XRT_DEV_COMPONENT}_PACKAGE_REQUIRES "<requirements>")`

I thought it was worth clarifying that you can only specify `"xrt-dev"`.

I also thought it was worth making `XRT_DEV_COMPONENT` a cached variable so that you could add it to the initial cmake options and the build would remember it for subsequent steps.  I can understand if you want to remove the caching and have this be a 'hidden' and unsupported feature of the build.